### PR TITLE
 initialize project with components

### PR DESCRIPTION
**Pull request by george827**
### **highlights**

- [ ] Initialize React app.
- [ ] Add React Router for the app's navigation:
- [ ] Books as the default view

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                <option value="Action">Action</option>
+                <option value="Science Fiction">Science Fiction</option>
+                <option value="Economy">Economy</option>
+              </select>
+            </div>
+            <div className="submit-btn">
+              <button type="submit">Add book</button>
+            </div>
+          </form>
+        </div>
```
Comment (FrederickMih) [inline src/components/Books.js]: - [ ] @george827, React was built on the concept to break pages into components. The more you break the page into small components, the more it becomes easier to use and scalable. This should have been another component called createNewBook. This component should render the form and should have the submit button. Let's kindly rectify this issue.


- [ ] You'll also need the `BookList` that does the following:
- Renders a list of books using the `Book` component
- Renders a form by using the CreateNewBook component

File: src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <h1>Under construction</h1>
```
Comment (FrederickMih) [inline src/components/Categories.js]: - [ ] Based on the requirements of this project, this component should display only a button with the text "Check status". Please kindly fix 🔧 it and move a step toward approval 👍

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi 🙋 @george827 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🟢 

- PR is professional 👍 
- All Linter checks passed 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @george827 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Created reusable components ✔️ 
- Installed `react-router-dom` ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**